test/ruby_lsp_rails/runner_client_test.rb

```diff
@@ -0,0 +1,51 @@
+# typed: true
+# frozen_string_literal: true
+
+require "test_helper"
+require "ruby_lsp/ruby_lsp_rails/runner_client"
+
+module RubyLsp
+  module Rails
+    class RunnerClientTest < ActiveSupport::TestCase
+      setup do
+        @client = T.let(RunnerClient.new, RunnerClient)
+      end
+
+      teardown do
+        @client.shutdown
+
+        assert_predicate @client.instance_variable_get(:@stdin), :closed?
+        assert_predicate @client.instance_variable_get(:@stdout), :closed?
+        assert_predicate @client.instance_variable_get(:@stderr), :closed?
+        refute_predicate @client.instance_variable_get(:@wait_thread), :alive?
+      end
+
+      test "#model returns information for the requested model" do
+        # test/dummy/db/migrate/20230330202955_create_users.rb
```

Comment:
✂️

Reply:
I've updated the comment - I wanted to make it clear where the column names used in that test are coming.